chains.yaml

```diff
@@ -1,5 +1,6 @@
 - name: noble-forwarding-simd
   dockerfile: cosmos
   build-target: make build
+  build-dir: simapp
```

Comment:
@boojamya Is this a required option in a new version of Heighliner?

Reply:
No, it's not. I just thought this was more straight forward. 

Right now, `make local-image` runs heighliner build, which checks the `chains.yaml` file, which runs `make build` (using root Makerfile), which cd's into simapp and then runs the simapp `make build`.

Adding `build-dir` will streamline this and heighliner will just run `make build` from the simapp folder. 

Happy to revert if you would like to keep original flow.